facade/baseFacade.go

```diff
@@ -500,3 +500,8 @@ func (epf *ElrondProxyFacade) GetTriesStatistics(shardID uint32) (*data.TrieStat
 func (epf *ElrondProxyFacade) GetEpochStartData(epoch uint32, shardID uint32) (*data.GenericAPIResponse, error) {
 	return epf.nodeStatusProc.GetEpochStartData(epoch, shardID)
 }
+
+// GetInternalStartOfEpochValidatorsInfo retrieves the validators info by nonce for a given shard
```

Comment:
wrong comment: the results are not for a given shard, but for the entire network, fetched from metachain nodes

Reply:
done